### PR TITLE
Review README, add note about symbiflow-examples

### PR DESCRIPTION
**Pull request by mgielda**
Signed-off-by: Michael Gielda <mgielda@antmicro.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # SymbiFlow Architecture Definitions
 
+**If you are looking for instructions how to use SymbiFlow, take a look at the [symbiflow-examples repository](https://github.com/SymbiFlow/symbiflow-examples).**
```
Comment (mithro) [inline README.md:3]: I think this needs to be more along the lines of "This repository is used during the development of architecture support in SymbiFlow, if you are looking to use the toolchain you should start at symbiflow-examples repository".

Comment (mgielda) [reply README.md]: OK, done!

**Comment by mgielda** [comment]
OK @mithro , included your suggestion.

**Review by mithro — APPROVED** [review]
LGTM.

**Pull request merged**